js/eslint.config.js

```diff
@@ -0,0 +1,226 @@
+const globals = require('globals');
+const { globalIgnores } = require('eslint/config')
+
+module.exports = [globalIgnores(["./*.js", "!./privatebin.js"]), {
+    languageOptions: {
+        globals: {
+            ...globals.amd,
+            ...globals.browser,
+            ...globals.jquery,
+            ...globals.node,
+            ...globals.mocha,
+            DOMPurify: "readonly",
+            cleanup: "writable",
+            describe: "readonly",
+            jsc: "readonly",
+            jsdom: "writable",
+            kjua: "writable",
+            WebCrypto: "writable",
+        },
+
+        // async & await are ECMAScript 2017 features, unicode character class escape are ECMAScript 2018 features
+        ecmaVersion: 2018,
+        sourceType: "commonjs",
```

Comment:
This is likely the reason `require` is needed, I guess we can switch to `import` back if we change it. Given `require` and CommonJS modules are really old...


But ah I see, then our codebase also would be written there? Then #365 would likely be the task to modernize it?

Reply:
Yes, we would need to switch to ES modules / .mjs files incl. this config all at once, for ESlint to work correctly.